src/Bicep.Cli/Commands/BuildParamsCommand.cs

```diff
@@ -54,10 +67,49 @@ public async Task<int> RunAsync(BuildParamsArguments args)
                 return 1;
             }
 
+            var paramsOverrides = Environment.GetEnvironmentVariable("BICEP_PARAMETERS_OVERRIDES");
+
+            if (paramsOverrides is { })
+            {
+                var paramsOverridesJson = JObject.Parse(paramsOverrides);
```

Comment:
Need to make sure we're ignoring date strings as Newtonsoft will attempt to treat these as special values by default. See #11675 for example.

Here's how we handle this for decompilation:
https://github.com/Azure/bicep/blob/65a5deeddb10657cd46e60c1700a4eaf3adfc749/src/Bicep.Decompiler/ArmHelpers/JTokenHelpers.cs#L23